### DISCUSSION TITLE
Accessing useQuery inputs in a msw handler

### DISCUSSION THREAD

**Discussion by sibteali786** [mswjs/msw#2238] [Q&A] [answered]
I am trying to access inputs to trpc useQuery in msw handler 
```jsx
 const { isLoading, isError, data } = trpc.topTraders.useQuery({
    count: maxCount,
    timeframe: selectedTopTradersPeriod,
  });
  ```
  
  and as per docs these are the few thing i found but they return nothing/ undefined 
  ```jsx
  http.get(`${baseUrl}/topTraders`, ({ request }) => {
          const url = new URL(request.url);
          console.log("Intercepted request for traders list", url);
          console.log("Params", url.searchParams.get("count"));
})
```
is there any way we can get access to count and timeframe values here ? 
Appreciate the help.

**Comment by kettanaito** [comment, marked as answer]
Hi. You access those values from the same place where they are being set in your `useQuery` hook. 

It can be either [query parameters](https://mswjs.io/docs/recipes/query-parameters) or in the [request body](https://mswjs.io/docs/network-behavior/rest#reading-request-body). Consult your request library's docs to learn where the `count` and `timeframe` values are being set.
